src/containers/LocationPage/LocationPage.jsx

```diff
@@ -433,8 +435,8 @@ class LocationPage extends PureComponent {
 
     // Create ASN Number to ISP Object dictionary
     const asnToISPObj = {};
-    for (const currIsp in topClientIsps) {
-      asnToISPObj[topClientIsps[currIsp].client_asn_number] = topClientIsps[currIsp];
+    for (let currIspIndex = 0; currIspIndex < topClientIsps.length; currIspIndex++) {
```

Comment:
Perhaps change this variable name to ispIndex so that it is a little shorter and more consistent with other variables like the asnIndex!

Reply:
Agreed. Also I commend you for fixing all these for loops' syntax!